packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+import { SearchBar } from './SearchBar';
+import { SearchFeedback } from './SearchFeedback';
+
+interface SearchContainerProps {
+  children: ReactNode;
+  onSubmit(event: React.MouseEvent, value: string): void;
+}
+
+export function SearchContainer({
+  children,
+  onSubmit,
+}: SearchContainerProps): React.ReactElement {
+  return (
+    <main className="m-auto w-full">
+      <div className="grid grid-cols-1 laptop:grid-cols-3 gap-y-6 pt-8 m-auto max-w-screen-laptop">
```

Comment:
@rebelchris after checking, the grid seems to be necessary for the result's UI.

Reply:
Ah right!! Yes remember now. But its static cols so should be fine then 🤘